data/events/scripts/player.lua

```diff
@@ -86,6 +86,13 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	if fromPosition.x ~= CONTAINER_POSITION and toPosition.x ~= CONTAINER_POSITION then
+		if not Tile(toPosition):getHouse() and item:getAttribute("wrapid") then
```

Comment:
I tested it and "not" prevents the player from moving the item.

Reply:
yes, you shouldn't be able to move an item that have wrapit to a normal tile, only house tile

Reply:
The problem is that I tested it with legs and armor.

Reply:
okay fixed, thanks